src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+  const h2 = document.createElement('h2');
```

Comment:
```suggestion
  const notification = document.createElement('div');
  const notificationTitle = document.createElement('h2');
```
it's better to name variables by meaning, not tag name